<?xml version="1.0"?>
<launch>

  <!-- Run safety check to reconstruct collision objets in the virtual enviroment model -->
  <node name="collision_check" pkg="obstacle_avoidance" type="collision_check" output="screen"> 
    <param name="collision_threshold" type="double" value="0.3"/>
    <param name="goal_threshold" type="double" value="0.05"/>

    <param name="cartesian_dmp_speed_scale" type="double" value="0.05"/>
    <param name="cartesian_dmp_dt" type="double" value="0.01"/>
    <param name="cartesain_dmp_gamma" type="double" value="800.0"/>
    <param name="cartesain_dmp_beta" type="double" value="6.68"/>
    <param name="cartesain_dmp_k" type="double" value="100"/>
    <param name="cartesain_dmp_d" type="double" value="20"/>
    <param name="cartesian_dmp_n_weights_per_dim" type="int" value="100"/>
    <param name="cartesian_dmp_n_bases" type="int" value="100"/>
    <param name="ik_jump_threshold_factor" type="double" value="0.02"/>

  </node>

</launch>
